common/spec/dependabot/metadata_finders/base/release_finder_spec.rb

```diff
@@ -115,8 +115,8 @@
               expect(subject).
                 to eq(
                   "## v1.8.0\n"\
-                  "- Add 2018-2027 TARGET holiday defintions\n"\
-                  "- Add 2018-2027 Bankgirot holiday defintions"
```

Comment:
👋 Looks like there's still some old spellings, there's a spec failure expecting `defintions`, could you do a grep and change all of them?

Reply:
Afaict, the failures are from a foreign repository. I don't think I can fix this. My plan is to drop the commit and leave it for later.

Reply:
I've moved that commit and one other to: https://github.com/dependabot/dependabot-core/compare/master...jsoref:spelling-foreign-repo?expand=1